images/test-image/run.sh

```diff
@@ -10,4 +10,4 @@ command="${@:3}"
 
 git clone "https://github.com/${github_org}/${github_repo}"
 cd "${github_repo}"
-eval "${command}" || { cleanup_dind ; exit 1; }
+eval "${command}" || { cleanup_dind ; exit 1; }
```

Comment:
I think we should add `cleanup_dind` after this as well to stop docker if ${command} returns 0

Reply:
Done